From 3cf3f7fe78563ed8760c2ca14e42e4ea840c9989 Mon Sep 17 00:00:00 2001
From: "Yaoming.Qin" <B10945@freescale.com>
Date: Thu, 26 May 2011 00:59:02 +0800
Subject: [PATCH 156/486] ENGR00144027  Camera cts cases failed

The root cause is that for the new sensor, not all resolution is
supported. Update it, and all are fixed.

Signed-off-by: Yaoming Qin<B10945@freescale.com>
---
 libcamera/V4l2CsiDevice.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libcamera/V4l2CsiDevice.cpp b/libcamera/V4l2CsiDevice.cpp
index e391e19..e67f301 100755
--- a/libcamera/V4l2CsiDevice.cpp
+++ b/libcamera/V4l2CsiDevice.cpp
@@ -278,6 +278,8 @@ namespace android{
                 capturemode = 0;	/* VGA mode */
             else if (capturewidth == 320 && captureheight == 240)
                 capturemode = 1;	/* QVGA mode */
+            else if (capturewidth == 720 && captureheight == 480)
+                capturemode = 2;	/* PAL mode */
             else if (capturewidth == 720 && captureheight == 576)
                 capturemode = 3;	/* PAL mode */
             else if (capturewidth == 1280 && captureheight == 720)
-- 
1.8.0

